app/upgrades/v12/upgrades.go

```diff
@@ -83,6 +84,7 @@ func CreateUpgradeHandler(
 
 		// Set TWAP parameters to default values.
 		keepers.TwapKeeper.SetParams(ctx, twaptypes.DefaultParams())
+		keepers.LockupKeeper.SetParams(ctx, lockuptypes.DefaultParams())
```

Comment:
We should probably make a v13 upgrade handler and add it there

Reply:
good catch! just created a v13 upgrade handler